pkg/doc/did/doc_test.go

```diff
@@ -445,10 +445,8 @@ func TestPopulateKeyAgreements(t *testing.T) {
 		require.NoError(t, err)
 
 		_, err = ParseDocument(bytes)
-		require.Error(t, err)
-
-		expected := fmt.Sprintf("key %s not exist in did doc verification method", missingPubKeyID)
-		require.Contains(t, err.Error(), expected)
+		require.EqualError(t, err, fmt.Sprintf("populate key agreements failed: key %s not exist in did doc"+
```

Comment:
Looks like this isn't something you changed in this PR, but figured it's worth mentioning anyway: should be `does not exist` instead of `not exist`

Reply:
will update :)

Reply:
updated